src/Interpreters/ProcessorsProfileLog.cpp

```diff
@@ -27,9 +27,9 @@ ColumnsDescription ProcessorProfileLogElement::getColumnsDescription()
         {"event_time", std::make_shared<DataTypeDateTime>(), "The date and time when the event happened."},
         {"event_time_microseconds", std::make_shared<DataTypeDateTime64>(6), "The date and time with microseconds precision when the event happened."},
 
-        {"id", std::make_shared<DataTypeUInt64>(), "ID of processor."},
-        {"parent_ids", std::make_shared<DataTypeArray>(std::make_shared<DataTypeUInt64>()), "Parent processors IDs."},
-        {"plan_step", std::make_shared<DataTypeUInt64>(), "ID of the query plan step which created this processor. The value is zero if the processor was not added from any step."},
+        {"id", std::make_shared<DataTypeString>(), "ID of processor."},
```

Comment:
this log is stored on disk, I don't think we can just change data types. maybe we could just introduce a new field

Reply:
If we need compatibility for old version, add a new field is a good choice.
But I didn't have a good name to describe this except for id, maybe I add a field named `uniq_id` and `uniq_step_id` ?

Reply:
yeah, if systemlog detect schema not match, it will rename old table as tablename_index and create new one.
Change type may effect some tools that use this field, create new field may not break the rules.